Lib/test/test_asyncio/test_base_events.py

```diff
@@ -145,6 +145,58 @@ def test_ipaddr_info_no_inet_pton(self, m_socket):
                                                    socket.SOCK_STREAM,
                                                    socket.IPPROTO_TCP))
 
+    def test_interleave_ipaddrs(self):
+        addrinfos = [
```

Comment:
These tests would be a lot more readable if you gave each line a short name, so you could do e.g.
```py
A = ...
B = ...
...
addrinfos = [A, B, C, D, E, F, G, H]
expected - [A, E, B, F, C, G, D, H]
```
The definitions of A-H can be shared between the two tests.

Reply:
Done. I gave them names SIX_A, FOUR_A, etc to distinguish between IPv6 and IPv4 addresses.